GOVERNANCE.md

```diff
@@ -122,9 +122,21 @@ These members will be elected for a term of 1 year as follows:
 
 ## Approving and Onboarding Regular Members
 
-Regular members can self-nominate by opening a PR to add themselves to the Regular member list in the [README.md][README]. The PR should include an indication of which project they have been active in for 3 months as per the [CPC charter][cpc charter regular members].
+While our goal in the OpenJS Foundation is to do as much or our work in the public, on occasion, there are private matters and private materials. Voting and Regular members are included in these private matters and have access to private materials. Therefore, we have some requirements in place to ensure a level of involvement in our work and to hold an expectation of professionalism.
```

Comment:
This language may be improved. I'm open to suggestions

Reply:
maybe wordy? something like

“The CPC strives for transparency and performs nearly all of its work in public. Occasionally, sensitive issues arise and must be addressed privately. Only CPC members are privy to these matters. The requirements above help establish a baseline of trust and professionalism.”

Reply:
I'm taking @dmethvin's version. Thanks!